lib/llm/src/model_card/model.rs

```diff
@@ -125,12 +125,6 @@ pub struct ModelDeploymentCard {
     /// Incrementing count of how many times we published this card
     #[serde(default, skip_serializing)]
     pub revision: u64,
-
-    /// Does this model expect preprocessing (tokenization, etc) to be already done?
-    /// If this is true they get a BackendInput JSON. If this is false they get
-    /// an NvCreateChatCompletionRequest JSON.
-    #[serde(default)]
-    pub requires_preprocessing: bool,
```

Comment:
What's the equivalent signal/concept for a model needing or not needing pre-processing with this removed?

Reply:
It's now in `ModelType`. `ModelType::Backend` needs Dynamo to do pre-processing, `Chat` and `Completion` don't.